public/css/style.css

```diff
@@ -0,0 +1,409 @@
+/* HTML Elements settings */
```

Comment:
there is A LOT of hardcoded sizes here. Everywhere we've got hardcoded `48px` etc. Such things need to be refactored to variables. Please use less here (http://lesscss.org/). Its very simple, but allows you to use CSS variables and color transformations instead of hardcoding them. I'm almost sure this is the reason why we are constantly adjusting spacing here and spacing there. Without creating config file with variables everything will always be inconsistent. This one is very important.